_studio/mfx_lib/encode_hw/hevc/linux/g12/hevcehw_g12_scc_lin.h

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2020 Intel Corporation
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#pragma once
+
+#include "mfx_common.h"
+#if defined(MFX_ENABLE_HEVCE_SCC) && defined (MFX_VA_LINUX)
```

Comment:
I don't see you define `MFX_ENABLE_HEVCE_SCC` anywhere in this code. And it does not present in the current code base either:
```
$ fgrep -rsn MFX_ENABLE_HEVCE_SCC _studio/
$
```

This makes me think you submit non-verified change. Do you have sample_encode command line to show this change is functional?

Reply:
Hi. Thanks for pointing out. Sorry, I made a mistake regarding code base which introduces this macro. Have removed it. Yes, I have a command line proved functional with the latest commit. Command Line:  
 ./sample_encode h265 -i in.yuv -o out.bin  -qpi 17 -qpp 17 -qpb 17 -qsv-ff -hw -f 30 -w 480 -h 272 -CodecProfile 9 -g 10000 -r 1 -u 4 -n 20